test/controllers/EvidenceQuestionController.test.ts

```diff
@@ -0,0 +1,84 @@
+import 'reflect-metadata';
+
+import { expect } from 'chai';
+import { MOVED_TEMPORARILY, OK, UNPROCESSABLE_ENTITY } from 'http-status-codes';
+import request from 'supertest';
+
+import 'app/controllers/EvidenceRemovalController';
+import { Appeal } from 'app/models/Appeal';
+import { YesNo } from 'app/models/fields/YesNo';
+import { EVIDENCE_QUESTION_URI} from 'app/utils/Paths';
+
+import { createApp } from 'test/ApplicationFactory';
+
+const createAppeal = (): Appeal => {
```

Comment:
Any reason it is method not object literal? It is not parametrised, remains the same so I could fixed object literal.